src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import socialMedia from '../../assets/socialMedia.svg';
```

Comment:
After library imports (like `react`) enter a new line and then import custom stuff. This will be a differentiator between library and custom imports